packages/user/Host/plugin/crypto/src/lib.rs

```diff
@@ -0,0 +1,114 @@
+#[allow(warnings)]
+mod bindings;
+use bindings::*;
+
+mod errors;
+use errors::ErrorType::*;
+mod types;
+use types::*;
+
+use bindings::exports::host::crypto::types::{Keypair, Pem};
+use bindings::host::types::types as HostTypes;
+use bindings::supervisor::bridge::intf as Supervisor;
+use exports::host::crypto::keyvault::Guest as KeyVault;
+
+use trust::*;
+
+// Third-party crates
+use p256::ecdsa::{SigningKey, VerifyingKey};
+use p256::pkcs8::{DecodePrivateKey, EncodePrivateKey, EncodePublicKey, LineEnding};
+use rand_core::OsRng;
+
+psibase::define_trust! {
+    descriptions {
+        None => "
+        No trust grants these abilities:
+            - Create new keypairs
+            - Get the public key for a given private key
+            - Convert PEM to DER format
+        ",
```

Comment:
No description needed for trust=`None`, because things that require no trust don't trigger any user prompt.